{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"PharmacoGx.ipynb","provenance":[],"authorship_tag":"ABX9TyMkLHsuzFOsw//9S6dqeDrv"},"kernelspec":{"name":"ir","display_name":"R"},"language_info":{"name":"R"}},"cells":[{"cell_type":"code","metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"-6-ZYicdCe2W","executionInfo":{"status":"ok","timestamp":1619673114876,"user_tz":-480,"elapsed":4084,"user":{"displayName":"Indrik Wijaya","photoUrl":"","userId":"00208067985282844199"}},"outputId":"ae739a78-04ed-4326-b2b0-e1a12feed474"},"source":["library(tidyverse)"],"execution_count":null,"outputs":[{"output_type":"stream","text":["Warning message in system(\"timedatectl\", intern = TRUE):\n","“running command 'timedatectl' had status 1”\n","── \u001b[1mAttaching packages\u001b[22m ─────────────────────────────────────── tidyverse 1.3.1 ──\n","\n","\u001b[32m✔\u001b[39m \u001b[34mggplot2\u001b[39m 3.3.3     \u001b[32m✔\u001b[39m \u001b[34mpurrr  \u001b[39m 0.3.4\n","\u001b[32m✔\u001b[39m \u001b[34mtibble \u001b[39m 3.1.1     \u001b[32m✔\u001b[39m \u001b[34mdplyr  \u001b[39m 1.0.5\n","\u001b[32m✔\u001b[39m \u001b[34mtidyr  \u001b[39m 1.1.3     \u001b[32m✔\u001b[39m \u001b[34mstringr\u001b[39m 1.4.0\n","\u001b[32m✔\u001b[39m \u001b[34mreadr  \u001b[39m 1.4.0     \u001b[32m✔\u001b[39m \u001b[34mforcats\u001b[39m 0.5.1\n","\n","── \u001b[1mConflicts\u001b[22m ────────────────────────────────────────── tidyverse_conflicts() ──\n","\u001b[31m✖\u001b[39m \u001b[34mdplyr\u001b[39m::\u001b[32mfilter()\u001b[39m masks \u001b[34mstats\u001b[39m::filter()\n","\u001b[31m✖\u001b[39m \u001b[34mdplyr\u001b[39m::\u001b[32mlag()\u001b[39m    masks \u001b[34mstats\u001b[39m::lag()\n","\n"],"name":"stderr"}]},{"cell_type":"code","metadata":{"id":"turASPSaCiP4"},"source":["install.packages(\"BiocManager\")\n","BiocManager::install(c('hgu133a.db'))\n","install.packages(c('mRMRe','Hmisc', 'glmnet', 'caret','randomForest'))\n","install.packages(c('dqrng', 'sn'))\n","library(devtools)\n","devtools::install_github('bhklab/wCI', force = TRUE)\n","devtools::install_github(\"bhklab/PharmacoGx-ML\")"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"QtML4FR7ZmVe","executionInfo":{"status":"ok","timestamp":1632111131754,"user_tz":-480,"elapsed":4892,"user":{"displayName":"Indrik Wijaya","photoUrl":"https://lh3.googleusercontent.com/a/default-user=s64","userId":"00208067985282844199"}},"outputId":"cf562ddc-8a0e-4a81-eb41-2d22defd8ccf"},"source":["devtools::install_github(\"bhklab/mci\", ref='light')"],"execution_count":null,"outputs":[{"output_type":"stream","name":"stderr","text":["Downloading GitHub repo bhklab/mci@light\n","\n"]},{"output_type":"stream","name":"stdout","text":["\u001b[32m✔\u001b[39m  \u001b[90mchecking for file ‘/tmp/RtmpdctV8j/remotes4034fd769/bhklab-wCI-e8eba0b/DESCRIPTION’\u001b[39m\u001b[36m\u001b[39m\n","\u001b[90m─\u001b[39m\u001b[90m  \u001b[39m\u001b[90mpreparing ‘mCI’:\u001b[39m\u001b[36m\u001b[39m\n","\u001b[32m✔\u001b[39m  \u001b[90mchecking DESCRIPTION meta-information\u001b[39m\u001b[36m\u001b[39m\n","\u001b[90m─\u001b[39m\u001b[90m  \u001b[39m\u001b[90mchecking for LF line-endings in source and make files and shell scripts\u001b[39m\u001b[36m\u001b[39m\n","\u001b[90m─\u001b[39m\u001b[90m  \u001b[39m\u001b[90mchecking for empty or unneeded directories\u001b[39m\u001b[36m\u001b[39m\n","   Omitted ‘LazyData’ from DESCRIPTION\n","\u001b[90m─\u001b[39m\u001b[90m  \u001b[39m\u001b[90mbuilding ‘mCI_0.1.0.tar.gz’\u001b[39m\u001b[36m\u001b[39m\n","   \n"]},{"output_type":"stream","name":"stderr","text":["Installing package into ‘/usr/local/lib/R/site-library’\n","(as ‘lib’ is unspecified)\n","\n"]}]},{"cell_type":"code","metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"Nw9FV1LwV-zh","executionInfo":{"status":"ok","timestamp":1632110867592,"user_tz":-480,"elapsed":747373,"user":{"displayName":"Indrik Wijaya","photoUrl":"https://lh3.googleusercontent.com/a/default-user=s64","userId":"00208067985282844199"}},"outputId":"9e26e536-cf27-4c82-8657-5c438d70a353"},"source":["BiocManager::install('PharmacoGx')"],"execution_count":null,"outputs":[{"output_type":"stream","name":"stderr","text":["'getOption(\"repos\")' replaces Bioconductor standard repositories, see\n","'?repositories' for details\n","\n","replacement repositories:\n","    CRAN: https://cran.rstudio.com\n","\n","\n","Bioconductor version 3.13 (BiocManager 1.30.16), R 4.1.1 (2021-08-10)\n","\n","Installing package(s) 'PharmacoGx'\n","\n","also installing the dependencies ‘sass’, ‘lazyeval’, ‘gtools’, ‘slam’, ‘sets’, ‘limma’, ‘fastmatch’, ‘httpuv’, ‘xtable’, ‘sourcetools’, ‘later’, ‘promises’, ‘bslib’, ‘crosstalk’, ‘jquerylib’, ‘formatR’, ‘gplots’, ‘relations’, ‘marray’, ‘fgsea’, ‘shiny’, ‘DT’, ‘shinyjs’, ‘shinydashboard’, ‘visNetwork’, ‘SnowballC’, ‘lambda.r’, ‘futile.options’, ‘NISTunits’, ‘pracma’, ‘maps’, ‘MatrixGenerics’, ‘piano’, ‘lsa’, ‘GenomicRanges’, ‘DelayedArray’, ‘futile.logger’, ‘snow’, ‘celestial’, ‘plotrix’, ‘sm’, ‘mapproj’, ‘RANN’, ‘CoreGx’, ‘SummarizedExperiment’, ‘MultiAssayExperiment’, ‘BiocParallel’, ‘magicaxis’, ‘caTools’, ‘downloader’\n","\n","\n","Old packages: 'gert', 'openssl', 'rmarkdown', 'vroom', 'waldo', 'nlme'\n","\n"]}]},{"cell_type":"code","metadata":{"id":"acDgV12SRqE5"},"source":["suppressPackageStartupMessages({\n","  library(PharmacoGx, verbose=FALSE)\n","  library(mCI, verbose=FALSE)\n","  #library(PharmacoGxML, verbose=FALSE)\n","  library(Biobase, verbose=FALSE)\n","})"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"colab":{"base_uri":"https://localhost:8080/","height":49},"id":"P3An7WG2xy8F","executionInfo":{"status":"ok","timestamp":1632117432118,"user_tz":-480,"elapsed":529,"user":{"displayName":"Indrik Wijaya","photoUrl":"https://lh3.googleusercontent.com/a/default-user=s64","userId":"00208067985282844199"}},"outputId":"5ec21397-cac5-485a-d416-3452b8f2ebdb"},"source":["getOption('timeout')\n","options(timeout=1000)\n","getOption('timeout')"],"execution_count":15,"outputs":[{"output_type":"display_data","data":{"text/plain":["[1] 1000"],"text/latex":"1000","text/markdown":"1000","text/html":["1000"]},"metadata":{}},{"output_type":"display_data","data":{"text/plain":["[1] 1000"],"text/latex":"1000","text/markdown":"1000","text/html":["1000"]},"metadata":{}}]},{"cell_type":"code","metadata":{"colab":{"base_uri":"https://localhost:8080/","height":1000},"id":"Sz8-ZLmf0PrO","executionInfo":{"status":"ok","timestamp":1632118052504,"user_tz":-480,"elapsed":2084,"user":{"displayName":"Indrik Wijaya","photoUrl":"https://lh3.googleusercontent.com/a/default-user=s64","userId":"00208067985282844199"}},"outputId":"cd9953a2-9c55-4377-b78c-ae89144764b1"},"source":["availablePSets()"],"execution_count":18,"outputs":[{"output_type":"display_data","data":{"text/plain":["   Dataset Name Date Created             PSet Name         version     \n","1  CCLE         2020-06-24T14:39:26.588Z CCLE_2015         2015        \n","2  CTRPv2       2020-06-24T14:39:26.588Z CTRPv2_2015       2015        \n","3  FIMM         2020-06-24T14:39:26.588Z FIMM_2016         2016        \n","4  gCSI         2021-06-11T21:58:16.390Z gCSI_2019         2019        \n","5  GDSC         2020-06-24T14:39:26.588Z GDSC_2020(v2-8.2) 2020(v2-8.2)\n","6  GDSC         2020-06-24T14:39:26.588Z GDSC_2020(v1-8.2) 2020(v1-8.2)\n","7  GRAY         2021-02-23T14:39:26.588Z GRAY_2017         2017        \n","8  NCI60        2021-08-18T16:28:45.207Z NCI60_2021        2021        \n","9  PRISM        2021-08-18T16:28:45.207Z PRISM_2020        2020        \n","10 UHNBreast    2020-06-24T14:39:26.588Z UHNBreast_2019    2019        \n","   type       \n","1  sensitivity\n","2  sensitivity\n","3  sensitivity\n","4  NA         \n","5  sensitivity\n","6  sensitivity\n","7  sensitivity\n","8  sensitivity\n","9  sensitivity\n","10 both       \n","   publication                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       \n","1  Barretina, Jordi et al. “The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity.” Nature vol. 483,7391 603-7. 28 Mar. 2012, doi:10.1038/nature11003, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3320027/                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 \n","2  Rees, Matthew G et al. “Correlating chemical sensitivity and basal gene expression reveals mechanism of action.” Nature chemical biology vol. 12,2 (2016): 109-16. doi:10.1038/nchembio.1986    , Seashore-Ludlow, Brinton et al. “Harnessing Connectivity in a Large-Scale Small-Molecule Sensitivity Dataset.” Cancer discovery vol. 5,11 (2015): 1210-23. doi:10.1158/2159-8290.CD-15-0235     , Basu, Amrita et al. “An interactive resource to identify cancer genetic and lineage dependencies targeted by small molecules.” Cell vol. 154,5 (2013): 1151-1161. doi:10.1016/j.cell.2013.08.003, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4718762/                                                                                                                                           , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631646/                                                                                                                                           , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3954635/                                                                                                                                                                                                                                                                                                                   \n","3  Mpindi, J., Yadav, B., Östling, P. et al. Consistency in drug response profiling. Nature 540, E5–E6 (2016) doi:10.1038/nature20171, https://www.nature.com/articles/nature20171                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   \n","4  Petr Smirnov, Ian Smith, Zhaleh Safikhani, Wail Ba-alawi, Farnoosh Khodakarami, Eva Lin, Yihong Yu, Scott Martin, Janosch Ortmann, Tero Aittokallio, Marc Hafner, Benjamin Haibe-Kains. Evaluation of statistical approaches for association testing in noisy drug screening data. arXiv (2021), https://arxiv.org/abs/2104.14036                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 \n","5  Yang, Wanjuan et al. “Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.” Nucleic acids research vol. 41,Database issue (2013): D955-61. doi:10.1093/nar/gks1111, Iorio F, Knijnenburg TA, Vis DJ, et al. A Landscape of Pharmacogenomic Interactions in Cancer. Cell. 2016;166(3):740‐754. doi:10.1016/j.cell.2016.06.017                                                                    , Garnett MJ, Edelman EJ, Heidorn SJ, et al. Systematic identification of genomic markers of drug sensitivity in cancer cells. Nature. 2012;483(7391):570‐575. Published 2012 Mar 28. doi:10.1038/nature11005                 , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531057/                                                                                                                                                                       , https://pubmed.ncbi.nlm.nih.gov/27397505/                                                                                                                                                                                   , https://pubmed.ncbi.nlm.nih.gov/22460902/                                                                                                                                                                                   \n","6  Yang, Wanjuan et al. “Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.” Nucleic acids research vol. 41,Database issue (2013): D955-61. doi:10.1093/nar/gks1111, Iorio F, Knijnenburg TA, Vis DJ, et al. A Landscape of Pharmacogenomic Interactions in Cancer. Cell. 2016;166(3):740‐754. doi:10.1016/j.cell.2016.06.017                                                                    , Garnett MJ, Edelman EJ, Heidorn SJ, et al. Systematic identification of genomic markers of drug sensitivity in cancer cells. Nature. 2012;483(7391):570‐575. Published 2012 Mar 28. doi:10.1038/nature11005                 , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531057/                                                                                                                                                                       , https://pubmed.ncbi.nlm.nih.gov/27397505/                                                                                                                                                                                   , https://pubmed.ncbi.nlm.nih.gov/22460902/                                                                                                                                                                                   \n","7  Hafner, Marc et al. “Quantification of sensitivity and resistance of breast cancer cell lines to anti-cancer drugs using GR metrics.” Scientific data vol. 4 170166. 7 Nov. 2017, doi:10.1038/sdata.2017.166, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5674849/                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               \n","8  Shoemaker, R. The NCI60 human tumour cell line anticancer drug screen. Nat Rev Cancer 6, 813–823 (2006). https://doi.org/10.1038/nrc1951, https://www.nature.com/articles/nrc1951                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 \n","9  Corsello SM, Nagari RT, Spangler RD, et al. Discovering the anti-cancer potential of non-oncology drugs by systematic viability profiling. Nat Cancer. 2020;1(2):235-248. doi:10.1038/s43018-019-0018-6, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7328899/                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    \n","10 Mammoliti, Anthony et al. “Creating reproducible pharmacogenomic analysis pipelines.” Scientific data vol. 6,1 166. 3 Sep. 2019, doi:10.1038/s41597-019-0174-7, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6722117/                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             \n","   rawSeqDataRNA                                              \n","1  https://portal.gdc.cancer.gov/legacy-archive/search/f      \n","2                                                             \n","3  https://portal.gdc.cancer.gov/legacy-archive/search/f      \n","4  https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2706/\n","5  https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3983/\n","6  https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3983/\n","7  http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48213 \n","8  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32474\n","9                                                             \n","10 https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\n","   DOI                   \n","1  10.5281/zenodo.3905462\n","2  10.5281/zenodo.3905470\n","3  10.5281/zenodo.3905448\n","4  10.5281/zenodo.4737437\n","5  10.5281/zenodo.3905481\n","6  10.5281/zenodo.3905485\n","7  10.5281/zenodo.4557735\n","8  10.5281/zenodo.5417785\n","9  10.5281/zenodo.5296794\n","10 10.5281/zenodo.3905460\n","   Download                                                        \n","1  https://zenodo.org/record/3905462/files/CCLE.rds?download=1     \n","2  https://zenodo.org/record/3905470/files/CTRPv2.rds?download=1   \n","3  https://zenodo.org/record/3905448/files/FIMM.rds?download=1     \n","4  https://zenodo.org/record/4737437/files/gCSI2.rds?download=1    \n","5  https://zenodo.org/record/3905481/files/GDSC2.rds?download=1    \n","6  https://zenodo.org/record/3905485/files/GDSC1.rds?download=1    \n","7  https://zenodo.org/record/4557735/files/GRAY2017.rds?download=1 \n","8  https://zenodo.org/record/5417785/files/NCI60.rds?download=1    \n","9  https://zenodo.org/record/5296794/files/PRISM.rds?download=1    \n","10 https://zenodo.org/record/3905460/files/UHNBreast.rds?download=1"],"text/latex":"A data.frame: 10 × 9\n\\begin{tabular}{r|lllllllll}\n  & Dataset Name & Date Created & PSet Name & version & type & publication & rawSeqDataRNA & DOI & Download\\\\\n  & <chr> & <chr> & <chr> & <chr> & <chr> & <list> & <chr> & <chr> & <chr>\\\\\n\\hline\n\t1 & CCLE      & 2020-06-24T14:39:26.588Z & CCLE\\_2015         & 2015         & sensitivity & Barretina, Jordi et al. “The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity.” Nature vol. 483,7391 603-7. 28 Mar. 2012, doi:10.1038/nature11003, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3320027/                                                                                                                                      & https://portal.gdc.cancer.gov/legacy-archive/search/f       & 10.5281/zenodo.3905462 & https://zenodo.org/record/3905462/files/CCLE.rds?download=1     \\\\\n\t2 & CTRPv2    & 2020-06-24T14:39:26.588Z & CTRPv2\\_2015       & 2015         & sensitivity & Rees, Matthew G et al. “Correlating chemical sensitivity and basal gene expression reveals mechanism of action.” Nature chemical biology vol. 12,2 (2016): 109-16. doi:10.1038/nchembio.1986    , Seashore-Ludlow, Brinton et al. “Harnessing Connectivity in a Large-Scale Small-Molecule Sensitivity Dataset.” Cancer discovery vol. 5,11 (2015): 1210-23. doi:10.1158/2159-8290.CD-15-0235     , Basu, Amrita et al. “An interactive resource to identify cancer genetic and lineage dependencies targeted by small molecules.” Cell vol. 154,5 (2013): 1151-1161. doi:10.1016/j.cell.2013.08.003, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4718762/                                                                                                                                           , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631646/                                                                                                                                           , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3954635/                                                                                                                                            &                                                             & 10.5281/zenodo.3905470 & https://zenodo.org/record/3905470/files/CTRPv2.rds?download=1   \\\\\n\t3 & FIMM      & 2020-06-24T14:39:26.588Z & FIMM\\_2016         & 2016         & sensitivity & Mpindi, J., Yadav, B., Östling, P. et al. Consistency in drug response profiling. Nature 540, E5–E6 (2016) doi:10.1038/nature20171, https://www.nature.com/articles/nature20171                                                                                        & https://portal.gdc.cancer.gov/legacy-archive/search/f       & 10.5281/zenodo.3905448 & https://zenodo.org/record/3905448/files/FIMM.rds?download=1     \\\\\n\t4 & gCSI      & 2021-06-11T21:58:16.390Z & gCSI\\_2019         & 2019         & NA          & Petr Smirnov, Ian Smith, Zhaleh Safikhani, Wail Ba-alawi, Farnoosh Khodakarami, Eva Lin, Yihong Yu, Scott Martin, Janosch Ortmann, Tero Aittokallio, Marc Hafner, Benjamin Haibe-Kains. Evaluation of statistical approaches for association testing in noisy drug screening data. arXiv (2021), https://arxiv.org/abs/2104.14036                                                                                                                                                                                                                                                                & https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2706/ & 10.5281/zenodo.4737437 & https://zenodo.org/record/4737437/files/gCSI2.rds?download=1    \\\\\n\t5 & GDSC      & 2020-06-24T14:39:26.588Z & GDSC\\_2020(v2-8.2) & 2020(v2-8.2) & sensitivity & Yang, Wanjuan et al. “Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.” Nucleic acids research vol. 41,Database issue (2013): D955-61. doi:10.1093/nar/gks1111, Iorio F, Knijnenburg TA, Vis DJ, et al. A Landscape of Pharmacogenomic Interactions in Cancer. Cell. 2016;166(3):740‐754. doi:10.1016/j.cell.2016.06.017                                                                    , Garnett MJ, Edelman EJ, Heidorn SJ, et al. Systematic identification of genomic markers of drug sensitivity in cancer cells. Nature. 2012;483(7391):570‐575. Published 2012 Mar 28. doi:10.1038/nature11005                 , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531057/                                                                                                                                                                       , https://pubmed.ncbi.nlm.nih.gov/27397505/                                                                                                                                                                                   , https://pubmed.ncbi.nlm.nih.gov/22460902/                                                                                                                                                                                    & https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3983/ & 10.5281/zenodo.3905481 & https://zenodo.org/record/3905481/files/GDSC2.rds?download=1    \\\\\n\t6 & GDSC      & 2020-06-24T14:39:26.588Z & GDSC\\_2020(v1-8.2) & 2020(v1-8.2) & sensitivity & Yang, Wanjuan et al. “Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.” Nucleic acids research vol. 41,Database issue (2013): D955-61. doi:10.1093/nar/gks1111, Iorio F, Knijnenburg TA, Vis DJ, et al. A Landscape of Pharmacogenomic Interactions in Cancer. Cell. 2016;166(3):740‐754. doi:10.1016/j.cell.2016.06.017                                                                    , Garnett MJ, Edelman EJ, Heidorn SJ, et al. Systematic identification of genomic markers of drug sensitivity in cancer cells. Nature. 2012;483(7391):570‐575. Published 2012 Mar 28. doi:10.1038/nature11005                 , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531057/                                                                                                                                                                       , https://pubmed.ncbi.nlm.nih.gov/27397505/                                                                                                                                                                                   , https://pubmed.ncbi.nlm.nih.gov/22460902/                                                                                                                                                                                    & https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3983/ & 10.5281/zenodo.3905485 & https://zenodo.org/record/3905485/files/GDSC1.rds?download=1    \\\\\n\t7 & GRAY      & 2021-02-23T14:39:26.588Z & GRAY\\_2017         & 2017         & sensitivity & Hafner, Marc et al. “Quantification of sensitivity and resistance of breast cancer cell lines to anti-cancer drugs using GR metrics.” Scientific data vol. 4 170166. 7 Nov. 2017, doi:10.1038/sdata.2017.166, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5674849/                                                                                                                                                        & http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48213  & 10.5281/zenodo.4557735 & https://zenodo.org/record/4557735/files/GRAY2017.rds?download=1 \\\\\n\t8 & NCI60     & 2021-08-18T16:28:45.207Z & NCI60\\_2021        & 2021         & sensitivity & Shoemaker, R. The NCI60 human tumour cell line anticancer drug screen. Nat Rev Cancer 6, 813–823 (2006). https://doi.org/10.1038/nrc1951, https://www.nature.com/articles/nrc1951                                                                                                  & https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32474 & 10.5281/zenodo.5417785 & https://zenodo.org/record/5417785/files/NCI60.rds?download=1    \\\\\n\t9 & PRISM     & 2021-08-18T16:28:45.207Z & PRISM\\_2020        & 2020         & sensitivity & Corsello SM, Nagari RT, Spangler RD, et al. Discovering the anti-cancer potential of non-oncology drugs by systematic viability profiling. Nat Cancer. 2020;1(2):235-248. doi:10.1038/s43018-019-0018-6, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7328899/                                                                                                                                                   &                                                             & 10.5281/zenodo.5296794 & https://zenodo.org/record/5296794/files/PRISM.rds?download=1    \\\\\n\t10 & UHNBreast & 2020-06-24T14:39:26.588Z & UHNBreast\\_2019    & 2019         & both        & Mammoliti, Anthony et al. “Creating reproducible pharmacogenomic analysis pipelines.” Scientific data vol. 6,1 166. 3 Sep. 2019, doi:10.1038/s41597-019-0174-7, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6722117/                                                                                                          & https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526 & 10.5281/zenodo.3905460 & https://zenodo.org/record/3905460/files/UHNBreast.rds?download=1\\\\\n\\end{tabular}\n","text/markdown":"\nA data.frame: 10 × 9\n\n| <!--/--> | Dataset Name &lt;chr&gt; | Date Created &lt;chr&gt; | PSet Name &lt;chr&gt; | version &lt;chr&gt; | type &lt;chr&gt; | publication &lt;list&gt; | rawSeqDataRNA &lt;chr&gt; | DOI &lt;chr&gt; | Download &lt;chr&gt; |\n|---|---|---|---|---|---|---|---|---|---|\n| 1 | CCLE      | 2020-06-24T14:39:26.588Z | CCLE_2015         | 2015         | sensitivity | Barretina, Jordi et al. “The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity.” Nature vol. 483,7391 603-7. 28 Mar. 2012, doi:10.1038/nature11003, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3320027/                                                                                                                                      | https://portal.gdc.cancer.gov/legacy-archive/search/f       | 10.5281/zenodo.3905462 | https://zenodo.org/record/3905462/files/CCLE.rds?download=1      |\n| 2 | CTRPv2    | 2020-06-24T14:39:26.588Z | CTRPv2_2015       | 2015         | sensitivity | Rees, Matthew G et al. “Correlating chemical sensitivity and basal gene expression reveals mechanism of action.” Nature chemical biology vol. 12,2 (2016): 109-16. doi:10.1038/nchembio.1986    , Seashore-Ludlow, Brinton et al. “Harnessing Connectivity in a Large-Scale Small-Molecule Sensitivity Dataset.” Cancer discovery vol. 5,11 (2015): 1210-23. doi:10.1158/2159-8290.CD-15-0235     , Basu, Amrita et al. “An interactive resource to identify cancer genetic and lineage dependencies targeted by small molecules.” Cell vol. 154,5 (2013): 1151-1161. doi:10.1016/j.cell.2013.08.003, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4718762/                                                                                                                                           , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631646/                                                                                                                                           , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3954635/                                                                                                                                            | <!----> | 10.5281/zenodo.3905470 | https://zenodo.org/record/3905470/files/CTRPv2.rds?download=1    |\n| 3 | FIMM      | 2020-06-24T14:39:26.588Z | FIMM_2016         | 2016         | sensitivity | Mpindi, J., Yadav, B., Östling, P. et al. Consistency in drug response profiling. Nature 540, E5–E6 (2016) doi:10.1038/nature20171, https://www.nature.com/articles/nature20171                                                                                        | https://portal.gdc.cancer.gov/legacy-archive/search/f       | 10.5281/zenodo.3905448 | https://zenodo.org/record/3905448/files/FIMM.rds?download=1      |\n| 4 | gCSI      | 2021-06-11T21:58:16.390Z | gCSI_2019         | 2019         | NA          | Petr Smirnov, Ian Smith, Zhaleh Safikhani, Wail Ba-alawi, Farnoosh Khodakarami, Eva Lin, Yihong Yu, Scott Martin, Janosch Ortmann, Tero Aittokallio, Marc Hafner, Benjamin Haibe-Kains. Evaluation of statistical approaches for association testing in noisy drug screening data. arXiv (2021), https://arxiv.org/abs/2104.14036                                                                                                                                                                                                                                                                | https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2706/ | 10.5281/zenodo.4737437 | https://zenodo.org/record/4737437/files/gCSI2.rds?download=1     |\n| 5 | GDSC      | 2020-06-24T14:39:26.588Z | GDSC_2020(v2-8.2) | 2020(v2-8.2) | sensitivity | Yang, Wanjuan et al. “Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.” Nucleic acids research vol. 41,Database issue (2013): D955-61. doi:10.1093/nar/gks1111, Iorio F, Knijnenburg TA, Vis DJ, et al. A Landscape of Pharmacogenomic Interactions in Cancer. Cell. 2016;166(3):740‐754. doi:10.1016/j.cell.2016.06.017                                                                    , Garnett MJ, Edelman EJ, Heidorn SJ, et al. Systematic identification of genomic markers of drug sensitivity in cancer cells. Nature. 2012;483(7391):570‐575. Published 2012 Mar 28. doi:10.1038/nature11005                 , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531057/                                                                                                                                                                       , https://pubmed.ncbi.nlm.nih.gov/27397505/                                                                                                                                                                                   , https://pubmed.ncbi.nlm.nih.gov/22460902/                                                                                                                                                                                    | https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3983/ | 10.5281/zenodo.3905481 | https://zenodo.org/record/3905481/files/GDSC2.rds?download=1     |\n| 6 | GDSC      | 2020-06-24T14:39:26.588Z | GDSC_2020(v1-8.2) | 2020(v1-8.2) | sensitivity | Yang, Wanjuan et al. “Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.” Nucleic acids research vol. 41,Database issue (2013): D955-61. doi:10.1093/nar/gks1111, Iorio F, Knijnenburg TA, Vis DJ, et al. A Landscape of Pharmacogenomic Interactions in Cancer. Cell. 2016;166(3):740‐754. doi:10.1016/j.cell.2016.06.017                                                                    , Garnett MJ, Edelman EJ, Heidorn SJ, et al. Systematic identification of genomic markers of drug sensitivity in cancer cells. Nature. 2012;483(7391):570‐575. Published 2012 Mar 28. doi:10.1038/nature11005                 , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531057/                                                                                                                                                                       , https://pubmed.ncbi.nlm.nih.gov/27397505/                                                                                                                                                                                   , https://pubmed.ncbi.nlm.nih.gov/22460902/                                                                                                                                                                                    | https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3983/ | 10.5281/zenodo.3905485 | https://zenodo.org/record/3905485/files/GDSC1.rds?download=1     |\n| 7 | GRAY      | 2021-02-23T14:39:26.588Z | GRAY_2017         | 2017         | sensitivity | Hafner, Marc et al. “Quantification of sensitivity and resistance of breast cancer cell lines to anti-cancer drugs using GR metrics.” Scientific data vol. 4 170166. 7 Nov. 2017, doi:10.1038/sdata.2017.166, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5674849/                                                                                                                                                        | http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48213  | 10.5281/zenodo.4557735 | https://zenodo.org/record/4557735/files/GRAY2017.rds?download=1  |\n| 8 | NCI60     | 2021-08-18T16:28:45.207Z | NCI60_2021        | 2021         | sensitivity | Shoemaker, R. The NCI60 human tumour cell line anticancer drug screen. Nat Rev Cancer 6, 813–823 (2006). https://doi.org/10.1038/nrc1951, https://www.nature.com/articles/nrc1951                                                                                                  | https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32474 | 10.5281/zenodo.5417785 | https://zenodo.org/record/5417785/files/NCI60.rds?download=1     |\n| 9 | PRISM     | 2021-08-18T16:28:45.207Z | PRISM_2020        | 2020         | sensitivity | Corsello SM, Nagari RT, Spangler RD, et al. Discovering the anti-cancer potential of non-oncology drugs by systematic viability profiling. Nat Cancer. 2020;1(2):235-248. doi:10.1038/s43018-019-0018-6, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7328899/                                                                                                                                                   | <!----> | 10.5281/zenodo.5296794 | https://zenodo.org/record/5296794/files/PRISM.rds?download=1     |\n| 10 | UHNBreast | 2020-06-24T14:39:26.588Z | UHNBreast_2019    | 2019         | both        | Mammoliti, Anthony et al. “Creating reproducible pharmacogenomic analysis pipelines.” Scientific data vol. 6,1 166. 3 Sep. 2019, doi:10.1038/s41597-019-0174-7, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6722117/                                                                                                          | https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526 | 10.5281/zenodo.3905460 | https://zenodo.org/record/3905460/files/UHNBreast.rds?download=1 |\n\n","text/html":["<table class=\"dataframe\">\n","<caption>A data.frame: 10 × 9</caption>\n","<thead>\n","\t<tr><th></th><th scope=col>Dataset Name</th><th scope=col>Date Created</th><th scope=col>PSet Name</th><th scope=col>version</th><th scope=col>type</th><th scope=col>publication</th><th scope=col>rawSeqDataRNA</th><th scope=col>DOI</th><th scope=col>Download</th></tr>\n","\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;list&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n","</thead>\n","<tbody>\n","\t<tr><th scope=row>1</th><td>CCLE     </td><td>2020-06-24T14:39:26.588Z</td><td>CCLE_2015        </td><td>2015        </td><td>sensitivity</td><td>Barretina, Jordi et al. “The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity.” Nature vol. 483,7391 603-7. 28 Mar. 2012, doi:10.1038/nature11003, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3320027/                                                                                                                                     </td><td>https://portal.gdc.cancer.gov/legacy-archive/search/f      </td><td>10.5281/zenodo.3905462</td><td>https://zenodo.org/record/3905462/files/CCLE.rds?download=1     </td></tr>\n","\t<tr><th scope=row>2</th><td>CTRPv2   </td><td>2020-06-24T14:39:26.588Z</td><td>CTRPv2_2015      </td><td>2015        </td><td>sensitivity</td><td>Rees, Matthew G et al. “Correlating chemical sensitivity and basal gene expression reveals mechanism of action.” Nature chemical biology vol. 12,2 (2016): 109-16. doi:10.1038/nchembio.1986    , Seashore-Ludlow, Brinton et al. “Harnessing Connectivity in a Large-Scale Small-Molecule Sensitivity Dataset.” Cancer discovery vol. 5,11 (2015): 1210-23. doi:10.1158/2159-8290.CD-15-0235     , Basu, Amrita et al. “An interactive resource to identify cancer genetic and lineage dependencies targeted by small molecules.” Cell vol. 154,5 (2013): 1151-1161. doi:10.1016/j.cell.2013.08.003, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4718762/                                                                                                                                           , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631646/                                                                                                                                           , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3954635/                                                                                                                                           </td><td>                                                           </td><td>10.5281/zenodo.3905470</td><td>https://zenodo.org/record/3905470/files/CTRPv2.rds?download=1   </td></tr>\n","\t<tr><th scope=row>3</th><td>FIMM     </td><td>2020-06-24T14:39:26.588Z</td><td>FIMM_2016        </td><td>2016        </td><td>sensitivity</td><td>Mpindi, J., Yadav, B., Östling, P. et al. Consistency in drug response profiling. Nature 540, E5–E6 (2016) doi:10.1038/nature20171, https://www.nature.com/articles/nature20171                                                                                       </td><td>https://portal.gdc.cancer.gov/legacy-archive/search/f      </td><td>10.5281/zenodo.3905448</td><td>https://zenodo.org/record/3905448/files/FIMM.rds?download=1     </td></tr>\n","\t<tr><th scope=row>4</th><td>gCSI     </td><td>2021-06-11T21:58:16.390Z</td><td>gCSI_2019        </td><td>2019        </td><td>NA         </td><td>Petr Smirnov, Ian Smith, Zhaleh Safikhani, Wail Ba-alawi, Farnoosh Khodakarami, Eva Lin, Yihong Yu, Scott Martin, Janosch Ortmann, Tero Aittokallio, Marc Hafner, Benjamin Haibe-Kains. Evaluation of statistical approaches for association testing in noisy drug screening data. arXiv (2021), https://arxiv.org/abs/2104.14036                                                                                                                                                                                                                                                               </td><td>https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-2706/</td><td>10.5281/zenodo.4737437</td><td>https://zenodo.org/record/4737437/files/gCSI2.rds?download=1    </td></tr>\n","\t<tr><th scope=row>5</th><td>GDSC     </td><td>2020-06-24T14:39:26.588Z</td><td>GDSC_2020(v2-8.2)</td><td>2020(v2-8.2)</td><td>sensitivity</td><td>Yang, Wanjuan et al. “Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.” Nucleic acids research vol. 41,Database issue (2013): D955-61. doi:10.1093/nar/gks1111, Iorio F, Knijnenburg TA, Vis DJ, et al. A Landscape of Pharmacogenomic Interactions in Cancer. Cell. 2016;166(3):740‐754. doi:10.1016/j.cell.2016.06.017                                                                    , Garnett MJ, Edelman EJ, Heidorn SJ, et al. Systematic identification of genomic markers of drug sensitivity in cancer cells. Nature. 2012;483(7391):570‐575. Published 2012 Mar 28. doi:10.1038/nature11005                 , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531057/                                                                                                                                                                       , https://pubmed.ncbi.nlm.nih.gov/27397505/                                                                                                                                                                                   , https://pubmed.ncbi.nlm.nih.gov/22460902/                                                                                                                                                                                   </td><td>https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3983/</td><td>10.5281/zenodo.3905481</td><td>https://zenodo.org/record/3905481/files/GDSC2.rds?download=1    </td></tr>\n","\t<tr><th scope=row>6</th><td>GDSC     </td><td>2020-06-24T14:39:26.588Z</td><td>GDSC_2020(v1-8.2)</td><td>2020(v1-8.2)</td><td>sensitivity</td><td>Yang, Wanjuan et al. “Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells.” Nucleic acids research vol. 41,Database issue (2013): D955-61. doi:10.1093/nar/gks1111, Iorio F, Knijnenburg TA, Vis DJ, et al. A Landscape of Pharmacogenomic Interactions in Cancer. Cell. 2016;166(3):740‐754. doi:10.1016/j.cell.2016.06.017                                                                    , Garnett MJ, Edelman EJ, Heidorn SJ, et al. Systematic identification of genomic markers of drug sensitivity in cancer cells. Nature. 2012;483(7391):570‐575. Published 2012 Mar 28. doi:10.1038/nature11005                 , https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531057/                                                                                                                                                                       , https://pubmed.ncbi.nlm.nih.gov/27397505/                                                                                                                                                                                   , https://pubmed.ncbi.nlm.nih.gov/22460902/                                                                                                                                                                                   </td><td>https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-3983/</td><td>10.5281/zenodo.3905485</td><td>https://zenodo.org/record/3905485/files/GDSC1.rds?download=1    </td></tr>\n","\t<tr><th scope=row>7</th><td>GRAY     </td><td>2021-02-23T14:39:26.588Z</td><td>GRAY_2017        </td><td>2017        </td><td>sensitivity</td><td>Hafner, Marc et al. “Quantification of sensitivity and resistance of breast cancer cell lines to anti-cancer drugs using GR metrics.” Scientific data vol. 4 170166. 7 Nov. 2017, doi:10.1038/sdata.2017.166, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5674849/                                                                                                                                                       </td><td>http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48213 </td><td>10.5281/zenodo.4557735</td><td>https://zenodo.org/record/4557735/files/GRAY2017.rds?download=1 </td></tr>\n","\t<tr><th scope=row>8</th><td>NCI60    </td><td>2021-08-18T16:28:45.207Z</td><td>NCI60_2021       </td><td>2021        </td><td>sensitivity</td><td>Shoemaker, R. The NCI60 human tumour cell line anticancer drug screen. Nat Rev Cancer 6, 813–823 (2006). https://doi.org/10.1038/nrc1951, https://www.nature.com/articles/nrc1951                                                                                                 </td><td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32474</td><td>10.5281/zenodo.5417785</td><td>https://zenodo.org/record/5417785/files/NCI60.rds?download=1    </td></tr>\n","\t<tr><th scope=row>9</th><td>PRISM    </td><td>2021-08-18T16:28:45.207Z</td><td>PRISM_2020       </td><td>2020        </td><td>sensitivity</td><td>Corsello SM, Nagari RT, Spangler RD, et al. Discovering the anti-cancer potential of non-oncology drugs by systematic viability profiling. Nat Cancer. 2020;1(2):235-248. doi:10.1038/s43018-019-0018-6, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7328899/                                                                                                                                                  </td><td>                                                           </td><td>10.5281/zenodo.5296794</td><td>https://zenodo.org/record/5296794/files/PRISM.rds?download=1    </td></tr>\n","\t<tr><th scope=row>10</th><td>UHNBreast</td><td>2020-06-24T14:39:26.588Z</td><td>UHNBreast_2019   </td><td>2019        </td><td>both       </td><td>Mammoliti, Anthony et al. “Creating reproducible pharmacogenomic analysis pipelines.” Scientific data vol. 6,1 166. 3 Sep. 2019, doi:10.1038/s41597-019-0174-7, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6722117/                                                                                                         </td><td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526</td><td>10.5281/zenodo.3905460</td><td>https://zenodo.org/record/3905460/files/UHNBreast.rds?download=1</td></tr>\n","</tbody>\n","</table>\n"]},"metadata":{}}]},{"cell_type":"code","metadata":{"colab":{"base_uri":"https://localhost:8080/","height":257},"id":"XAA-Sif3R0t-","executionInfo":{"status":"error","timestamp":1632122583219,"user_tz":-480,"elapsed":657108,"user":{"displayName":"Indrik Wijaya","photoUrl":"https://lh3.googleusercontent.com/a/default-user=s64","userId":"00208067985282844199"}},"outputId":"1cdf830a-b8ef-46b8-d7c7-9634a8cdbb8c"},"source":["GDSC <- downloadPSet(\"GDSC_2020(v2-8.2)\") \n","CCLE <- downloadPSet(\"CCLE_2015\")"],"execution_count":20,"outputs":[{"output_type":"stream","name":"stderr","text":["Warning message in download.file(url, method = method, ...):\n","“downloaded length 740855808 != reported length 2949262528”\n","Warning message in download.file(url, method = method, ...):\n","“URL 'https://zenodo.org/record/3905462/files/CCLE.rds?download=1': Timeout of 600 seconds was reached”\n"]},{"output_type":"error","ename":"ERROR","evalue":"ignored","traceback":["Error in download.file(url, method = method, ...): download from 'https://zenodo.org/record/3905462/files/CCLE.rds?download=1' failed\nTraceback:\n","1. downloadPSet(\"CCLE_2015\")","2. downloader::download(url = as.character(pSetTable[whichx, \"Download\"]), \n .     destfile = file.path(saveDir, pSetFileName), quiet = !verbose, \n .     mode = \"wb\")","3. download.file(url, method = method, ...)"]}]},{"cell_type":"code","metadata":{"id":"wSzEVniXR5B5","executionInfo":{"status":"aborted","timestamp":1632117463815,"user_tz":-480,"elapsed":28198,"user":{"displayName":"Indrik Wijaya","photoUrl":"https://lh3.googleusercontent.com/a/default-user=s64","userId":"00208067985282844199"}}},"source":["mycol <- c(\"#8dd3c7\",\"#ffffb3\",\"#bebada\",\"#fb8072\",\"#80b1d3\",\"#fdb462\",\n","           \"#b3de69\",\"#fccde5\",\"#d9d9d9\",\"#bc80bd\",\"#ccebc5\",\"#ffed6f\",\n","           \"#a6cee3\",\"#1f78b4\",\"#b2df8a\",\"#33a02c\",\"#fb9a99\",\"#e31a1c\",\n","           \"#fdbf6f\",\"#ff7f00\",\"#cab2d6\",\"#6a3d9a\",\"#ffff99\",\"#b15928\")\n","pie(table(CCLE@cell[,\"tissueid\"]), \n","    col=mycol, \n","    main=\"Tissue types\", \n","    radius=1, \n","    cex=0.8)"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"id":"xnWIRF2eR7Rs"},"source":[""],"execution_count":null,"outputs":[]}]}